### DISCUSSION TITLE
Account has been blocked

### DISCUSSION THREAD

**Discussion by vladliviu358** [vercel/community#4205] [Account Issue] [answered]
### Question

Hi Vercel Team,

Haven't logged in in a while. Now I am getting the message that my account has been blocked. Please help
![image](https://github.com/vercel/community/assets/116064810/eab94439-ee79-4fd0-8934-2f5bf8cab4aa)

**Comment by maktubz1** [comment]
I'm getting the same error.

**Comment by multilinks50** [comment]
same case with me

**Comment by hungchihsueh** [comment]
same here please help!

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

You can email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with the support team. For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by multilinks50** [reply]
i am have send so much mail to registration@vercel.com but got no reply ..then how i will signup if no one is there to help?

**Reply by amyegan** [reply]
Tickets are processed by the order in which they are received. You can find more info about response times at https://vercel.com/guides/vercel-support-queue-time

**Reply by multilinks50** [reply]
Hey @amyegan, I still have not received any email, response or reply from Vercel. Is my problem going to be resolved or is my email going to remain blocked. Please help me out here, or ask someone to reach out since I have been left hanging since the moment i signed up on vercel

**Reply by amyegan** [reply]
If you let me know the case number, I can check on the ticket status for you.
